Adyen.Test/SerializerTest.cs

```diff
@@ -87,9 +87,91 @@ public void CheckoutSessionResponseCheckForIdTest()
             Assert.IsTrue(deserializedResponse.ToJson().Contains("\"id\": \"CS0068299CB8DA273A\","));
         }
 
+        [TestMethod]
+        public void EnsureSaleToPoiMessageSerializationDoesNotDependOnJsonConvertDefaultSettings()
+        {
+            var saleToPoiMessage = MockPosApiRequest.CreatePosPaymentRequest();
+            var serializedWithEmptyDefaultSettings = GetSerializedSaleToPoiMessage(saleToPoiMessage);
+
+            JsonConvert.DefaultSettings = () => new JsonSerializerSettings
+            {
+                ContractResolver = new DefaultContractResolver
+                {
+                    NamingStrategy = new CamelCaseNamingStrategy()
+                }
+            };
+
+            try
+            {
+                var serializedWithUpdatedDefaultSettings = GetSerializedSaleToPoiMessage(saleToPoiMessage);
+
+                Assert.AreEqual(serializedWithEmptyDefaultSettings, serializedWithUpdatedDefaultSettings);
+            }
+            finally
+            {
+                JsonConvert.DefaultSettings = null;
```

Comment:
Do we need need to set this to `null` afterwards? I think the the try/finally is redundant.

Reply:
Splitted.